src/components/comment/commentForm.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable react/forbid-prop-types */
+import React, { Component } from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { addComment } from '../../actions/comment/postComment';
+
+export class CommentForm extends Component {
+  constructor(props) {
```

Comment:
Remove the constructor and use arrow functions for the functions in this class to avoid binding